### PR TITLE
MGDOBR-1203:Disable processor creation for FAILED SE instances

### PR DESCRIPTION
**Pull request by ppadti**
JIRA: https://issues.redhat.com/browse/MGDOBR-1203

Steps to test it:

- Create a bridge instance with Incorrect parameters to enter into Failed state.
- Open the failed bridge instance detail page.
- Create processor button will be disabled for the failed bridge instance.

### CODE REVIEW

**Review by jomarko — COMMENTED** [review]
I am fine with changes, if we include a test https://github.com/ppadti/sandbox-ui/pull/3

**Review by jomarko — APPROVED** [review]
Thank you

File: src/app/Instance/InstancePage/ProcessorsTabContent.tsx [hunk 1]
```diff
@@ -142,6 +137,20 @@ export const ProcessorsTabContent = ({
     error: processorsError,
   } = useGetProcessorsApi();
 
+  const customToolbarElement = areProcessorsLoading ? (
+    <Skeleton width={"170px"} height={"35px"} />
+  ) : bridgeStatus == ManagedResourceStatus.Failed ? (
```
Comment (vpellegrino) [inline src/app/Instance/InstancePage/ProcessorsTabContent.tsx:142]: There is a tiny time interval when `bridgeStatus` is `undefined` and the "Create processor" button is enabled.
Maybe a "better" condition for enabling the processor creation could be `bridgeStatus !== ManagedResourceStatus.Ready`?

File: src/app/Instance/InstancePage/ProcessorsTabContent.tsx [hunk 2]
```diff
@@ -142,6 +137,20 @@ export const ProcessorsTabContent = ({
...
+    <Button ouiaId="create-processor" variant="primary" isDisabled={true}>
+      {t("processor.createProcessor")}
+    </Button>
+  ) : (
+    <Link to={`/instance/${instanceId}/create-processor`}>
+      <Button ouiaId="create-processor" variant="primary">
+        {t("processor.createProcessor")}
+      </Button>
+    </Link>
+  );
```
Comment (vpellegrino) [inline src/app/Instance/InstancePage/ProcessorsTabContent.tsx:152]: Generally speaking, I'm not too fond of using two chained ternary operators. Moreover, we can avoid repeating the whole `Button` definition with just one property (`isDisabled`) that could depend on a variable.

**Review by vpellegrino — COMMENTED** [review]
Thanks for this PR @ppadti 
I love the introduction of the Skeleton when the page is still loading.
Just left a couple of minor observations.

**Review by vpellegrino — APPROVED** [review]
LGTM, thank you @ppadti

File: src/app/Instance/InstancePage/ProcessorsTabContent.tsx [hunk 3]
```diff
@@ -142,6 +138,21 @@ export const ProcessorsTabContent = ({
     error: processorsError,
   } = useGetProcessorsApi();
 
+  const customToolbarElement = areProcessorsLoading ? (
+    <Skeleton width={"170px"} height={"35px"} />
+  ) : (
+    <Button
```
Comment (kelvah) [inline src/app/Instance/InstancePage/ProcessorsTabContent.tsx:144]: Why do we check on the processors loading status? In theory the "create processor" button depends only on the bridge status, we could display it as soon as it's available. Like `!bridgeStatus ? ... `
Wdyt @ppadti @vpellegrino?

Comment (vpellegrino) [reply src/app/Instance/InstancePage/ProcessorsTabContent.tsx:144]: You're right @kelvah. Button loading should depend on the bridge, and not on the list of processors (different API call).

**Review by kelvah — APPROVED** [review]
LGTM. Nice work @ppadti!

**Pull request merged**